pandasai/response/Context.py

```diff
@@ -0,0 +1,28 @@
+from pandasai.helpers.logger import Logger
+from pandasai.schemas.df_config import Config
+
+
+class Context:
+    _config = None
+    _logger = None
+    _engine: str = None
+
+    def __init__(self, config: Config, logger=Logger, engine: str = None) -> None:
```

Comment:
A typo here. Did you mean to use it as a type or as a default value? So far it's passing the class as a value

Reply:
Typo i fixed that in the new commit